src/sqlittaaja/gui.py

```diff
@@ -0,0 +1,127 @@
+from tkinter import Tk, ttk
+from tkinter.filedialog import askopenfilename
+import tomllib
+from sqlittaaja.checker import check_exercises
+from sqlittaaja.extractor import extract
+from sqlittaaja.diff_check import compute_similarity
+
+
+def load_config_file():
+    """Loads config file."""
+    global config_file_path
+
+    config_file_path = askopenfilename(filetypes=[("Toml files", "*.toml")])
+    if config_file_path:
+        load_config_btn.configure(text="Done!")
+
+
+def print_gui_score():
+    """Checks exercises and prints score to GUI."""
+
+    try:
+        with open(config_file_path, "rb") as file:
+            config = tomllib.load(file)
+
+            init_script = config["answer"].get("initialize", "")
+            answer = config["answer"]["exercise"]
+            exercises = extract(config["exercise"]["path"])
+
+            student_scores = check_exercises(init_script, answer, exercises)
+            answer_similarities = compute_similarity(exercises)
```

Comment:
There seems to be a lot of code duplication up untill this point. Do you think we could make these into a separate function?